gtk: replace deprecated gtk_type_new and _unique

henning-schild opened the pull request:
Calling these causes segfaults.

This fix is based on
https://launchpadlibrarian.net/295253441/gmrun-tdp-patch.diff

wdlkmpx committed:
clean up/update build system

wdlkmpx committed:
always use libc's system() function to run commands

it's the right way. the author himself didn't remember why he added
the other unnecessary option..

wdlkmpx committed:
configure.ac: remove unneded AM_PROG_AR

wdlkmpx committed:
bump version to 0.9.3

wdlkmpx committed:
use gtk_window_set_position()

wdlkmpx committed:
remove commented out code

wdlkmpx committed:
remove libpopt dep / --geometry param

wdlkmpx committed:
remove gmrun.spec.in...

wdlkmpx committed:
configure.ac: update AM_INIT_AUTOMAKE

wdlkmpx committed:
update .gitignore

wdlkmpx committed:
remove unused screen_contains_pointer()

wdlkmpx committed:
add 'make-release' script

wdlkmpx committed:
configure.ac: remove unused XQueryPointer stuff

wdlkmpx committed:
simplify / use @sysconfdir@ for gmrunrc

wdlkmpx committed:
bump version to 0.9.4w

henning-schild committed:
gtk: replace deprecated gtk_type_new and _unique

Calling these causes segfaults.

This fix is based on
https://launchpadlibrarian.net/295253441/gmrun-tdp-patch.diff

henning-schild commented:
@wdlkmpx you might want to have a look at this as well

The segfaults starting coming when i switched my system to gcc6.4, might have to do with pic and pie

wdlkmpx committed:
add .travis.yml

wdlkmpx commented:
Thanks.. https://github.com/wdlkmpx/gmrun/commit/6a0ea012a7f51fdc5639bedabe446c9035461cda

henning-schild committed:
gtk: do not select any text to keep clipboard clean

If you use gmrun to open an editor to drop you clipboard content in it,
your clipboard content will get messed up by gmrun.
